finetune/finetune.py

```diff
@@ -164,9 +163,10 @@ def get_finetune_config():
 
 
 def main(external_config = None):
-    config = get_finetune_config()
-    if external_config is not None:
-        config.merge(external_config)
+    if not external_config:
+        config = get_finetune_config()
+    else:
+        config = external_config
```

Comment:
Just curious, when do we use external_config? It seems it will never be used.

Reply:
https://github.com/intel/llm-on-ray/blob/cd05f0e36814f267c714c404fabb4ef5bac3df3f/inference/start_ui.py#L407 The main func in finetune.py is also used in ui finetune module, but we can't pass a yaml file to it.

Reply:
I see. Thanks for the explanation!